internal/lib/avi_api.go

```diff
@@ -66,10 +66,10 @@ func AviGet(client *clients.AviClient, uri string, response interface{}, retryNu
 		if aviError, ok := err.(session.AviError); ok && aviError.HttpStatusCode == 403 {
 			utils.AviLog.Debugf("Switching to admin context from %s", GetTenant())
 			SetAdminTenant := session.SetTenant(GetAdminTenant())
-			SetTenant := session.SetTenant(GetTenant())
 			SetAdminTenant(client.AviSession)
+			SetTenant := session.SetTenant(GetTenant())
```

Comment:
@arihantg What is the advantage of moving line from 69 to 70?

Reply:
just moved for ease in readability to group it with the defer statement on line 71